io/src/main/scala/fs2/io/file/pulls.scala

```diff
@@ -66,13 +66,13 @@ object pulls {
     * The `Pull` closes the provided `java.nio.channels.FileChannel` when it is done.
     */
   def fromFileChannel[F[_]](channel: F[FileChannel])(implicit F: Effect[F]): Pull[F, Nothing, FileHandle[F]] =
-    Pull.acquire(F.map(channel)(FileHandle.fromFileChannel[F]))(_.close())
+    Pull.acquire(F.map(channel)(FileHandle.fromFileChannel[F]))(_.close()).map { case (_, ch) => ch }
```

Comment:
kind of feels like we should keep `Pull.acquire` as is, and define it in terms of a new function, `Pull.acquireCancellable` (or some other name). Most uses of `Pull.acquire` are not going to use early termination.